scripts/development/m3_stack/m3aggregator.yml

```diff
@@ -74,58 +74,18 @@ http:
   readTimeout: 60s
   writeTimeout: 60s
 
-rawtcp:
-  listenAddress: 0.0.0.0:6000
-  keepAliveEnabled: true
-  keepAlivePeriod: 1m
-  retry:
-    initialBackoff: 5ms
-    backoffFactor: 2.0
-    maxBackoff: 1s
-    forever: true
-    jitter: true
-  readBufferSize: 1440
-  msgpackIterator:
-    ignoreHigherVersion: false
-    readerBufferSize: 1440
-    largeFloatsSize: 1024
-    largeFloatsPool:
-      buckets:
-        - count: 1024
-          capacity: 2048
-        - count: 512
-          capacity: 4096
-        - count: 256
-          capacity: 8192
-        - count: 128
-          capacity: 16384
-        - count: 64
-          capacity: 32768
-        - count: 32
-          capacity: 65536
-      watermark:
-        low: 0.001
-        high: 0.002
-  protobufIterator:
-    initBufferSize: 1440
-    maxMessageSize: 50000000  # max message size is 50MB
-    bytesPool:
-      buckets:
-        - count: 1024
-          capacity: 2048
-        - count: 512
-          capacity: 4096
-        - count: 256
-          capacity: 8192
-        - count: 128
-          capacity: 16384
-        - count: 64
-          capacity: 32768
-        - count: 32
-          capacity: 65536
+m3msg:
```

Comment:
Not for this PR, but we should add in some docs for these, and leave a few examples of rawtcp version (if someone needs it for whatever reason)

Reply:
Yup, agreed. Thankfully I kept both integration tests for the time being.